react/features/follow-me/reducer.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import {
+    SET_FOLLOW_ME_ACTIVE, SET_FOLLOW_ME_STATE
+} from './actionTypes';
+import { ReducerRegistry, set } from '../base/redux';
+
+/**
+ * Listen for actions that contain the Follow Me feature active state, so that it can be stored.
+ */
+ReducerRegistry.register(
+    'features/follow-me',
+    (state = {}, action) => {
+        switch (action.type) {
+
+        case SET_FOLLOW_ME_ACTIVE: {
+            let newState = set(state, 'followMeActive', action.enabled);
```

Comment:
naming nit: these keys can be stored without the "followMe" prefix because you're in the "follow-me" reducer key already.